jupyter-extension/jupyterlab_pachyderm/handlers.py

```diff
@@ -536,6 +516,29 @@ async def put(self, path):
             raise tornado.web.HTTPError(status_code=500, reason=repr(e))
         await self.finish()
 
+class MountPfsRepoHandler(BaseHandler):
+    @tornado.web.authenticated
+    async def put(self):
+        try:
+            body = self.get_json_body()
+            if not body:
+                raise ValueError("branch_uri does not exist")
```

Comment:
I think adding "in body of request" might be helpful context to these ValueError messages

Reply:
Will do agreed.